src/zeroD/ReactorBase.cpp

```diff
@@ -31,6 +37,18 @@ ReactorBase::~ReactorBase()
     }
 }
 
+bool ReactorBase::setDefaultName(const string& typ, int count)
+{
+    if (m_defaultNameSet) {
+        return false;
+    }
+    m_defaultNameSet = true;
+    if (m_name == "(none)" || m_name == "") {
+        m_name = fmt::format("{}_{}", typ, count);
+    }
+    return true;
```

Comment:
The logic between this and `ReactorNet::updateNames` results in the count being incremented even for reactors that already have names. Is this the intended behavior?

Reply:
This is indeed the intended behavior.